lib/pbench/server/database/models/users.py

```diff
@@ -28,7 +29,7 @@ class User(Database.Base):
     registered_on = Column(DateTime, nullable=False, default=datetime.datetime.now())
     email = Column(String(255), unique=True, nullable=False)
     role = Column(Enum(Roles), unique=False, nullable=True)
-    auth_tokens = relationship("ActiveTokens", backref="users")
+    auth_tokens = relationship("AuthToken", backref="users")
```

Comment:
I really hate to even ask, but...since we seem to be hung up on pluralization...why here is `auth_tokens` plural when `AuthToken` is not (and `users` _is_)?

Reply:
Because the `backref` is to the table not the object.

Reply:
And...?  `auth_tokens` is a reference to a table?  And, `"AuthToken"` is a reference to an object?

Reply:
The `auth_tokens` being assigned at line 32 is a reference to a column of the `users` table.  `AuthToken` is a SQLAlchemy object that refers to one row from the `auth_tokens` table.